src/python/bcc/__init__.py

```diff
@@ -651,7 +654,11 @@ def fix_syscall_fnname(self, name):
         name = _assert_is_bytes(name)
         for prefix in self._syscall_prefixes:
             if name.startswith(prefix):
-                return self.get_syscall_fnname(name[len(prefix):])
+                fnname = self.get_syscall_fnname(name[len(prefix):])
+                if fnname:
+                    return fnname
+                else:
+                    return name
```

Comment:
why changing this function? The tool did not use this function, right?

Reply:
I thought I should keep the logic as same as possible after changing get_syscall_fnname. But yes, this will be kept untouched in the later version.